usagov/usagov-2021
# Delete-old-templates: Delete old Template files

## Jira Task

No Jira ticket

## Description

These are currently unused templates from prior implementations that we had kept around incase we needed to roll back if there were issues with new implementation.

## Type of Changes

  - Add screenshot showing what it should look like
- [x] Other: Code Cleanup

## Testing Instructions


Make sure that homepage still has working cards in the all topics section and carousel.
Make sure that the navigation pages still have working cards.

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
